__tests__/functions.test.ts

```diff
@@ -89,6 +89,53 @@ describe('Testing compareLabels function', () => {
 })
 
 describe('Testing getIssueLabels function', () => {
+  const autoLabelTestCases = [
```

Comment:
seria interessante colocar esse codigo dentro de um `it('should `new` label', async () => {`, assim ficaria mais facil

Reply:
como to usando o [test.each](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout), cada caso vai gerar um novo it. eu escopei agora a variavel ao invez de deixar ela global, acho que ficou melhor. da uma olhada e me fala! abs